app/presenters/formats/place_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module Formats
+  class PlacePresenter < EditionFormatPresenter
+  private
+
+    def schema_name
+      'place'
+    end
+
+    def document_type
+      'place'
+    end
+
+    def details
+      details = { external_related_links: external_related_links }
+
+      details = details.merge(govspeak_fields)
+
+      if edition.place_type
+        details = details.merge(place_type: edition.place_type)
+      end
+
+      details
+    end
+
+    def govspeak_fields
+      %i(introduction more_information need_to_know).inject({}) do |details, field|
+        next details if edition[field].blank?
+
+        details[field] = [
+          {
+            content_type: "text/govspeak",
+            content: edition[field]
```

Comment:
I'm a bit confused by the `[]` operator here - is it common to access properties of the `Edition` model using the hash accessor?